autogpt_platform/backend/backend/blocks/basic.py

```diff
@@ -88,6 +88,29 @@ def run(self, input_data: Input, **kwargs) -> BlockOutput:
         yield "output", input_data.data or input_data.input
 
 
+class PrintToConsoleBlock(Block):
+    class Input(BlockSchema):
+        text: str = SchemaField(description="The text to print to the console.")
+
+    class Output(BlockSchema):
+        status: str = SchemaField(description="The status of the print operation.")
+
+    def __init__(self):
+        super().__init__(
+            id="f3b1c1b2-4c4f-4f0d-8d2f-4c4f0d8d2f4c",
+            description="Print the given text to the console, this is used for a debugging purpose.",
+            categories={BlockCategory.BASIC},
+            input_schema=PrintToConsoleBlock.Input,
+            output_schema=PrintToConsoleBlock.Output,
+            test_input={"text": "Hello, World!"},
+            test_output=("status", "printed"),
+        )
+
+    def run(self, input_data: Input, **kwargs) -> BlockOutput:
+        # print(">>>>> Print: ", input_data.text)
```

Comment:
You can:

change this to yield "output", text
and add output: str in Output

just to make it slightly more useful